todo-app/src/App.js

```diff
@@ -1,13 +1,21 @@
-import React, { Component } from 'react';
+import React from 'react';
+import TodoList from './TodoList';
+import Todos from './Todos';
 
-class App extends Component {
+export default class App extends React.Component {
+  renderTodos(Todos) {
+    return Todos.map((todo, index) => {
```

Comment:
Instead of using the `Todo` directly you would have set them in the state of the `App`. For example:

```js
this.state = {
  todos: Todos
};
```